src/examples/java/com/nylas/examples/other/EventsExample.java

```diff
@@ -117,7 +120,9 @@ protected static void basicEventCrud(Events events, Calendar primary) throws IOE
 	}
 
 	protected static void autocreateEvents(Events events, Calendar primary) throws IOException, RequestFailedException {
-		Event created = createBasicEvent(events, primary);
+		Event event = createBasicEvent(events, primary);
+		Event created = events.create(event, true);
+		System.out.println("Created: " + created);
```

Comment:
can this use logging? the log4j jar looks like it has been upgraded

Reply:
@jseller good call out! Will update events with logging and re-request your review 😄

Reply:
@jseller changes made to all examples and merge conflicts resolved.